### ISSUE TITLE
pytorch version

### ISSUE THREAD

**Issue by mihaifieraru** [kazuto1011/pspnet-pytorch#6]
Hi,

What pytorch version are you using? i'm using v0.2 and I get some errors that suggest I should use an older version.

Thanks

**Comment by zhijiew** [comment]
Could you please paste the error info? V0.3 will be better imo.

**Comment by mihaifieraru** [comment]
v0.2  had a different definition of softmax. 
I kept v0.2 and changed demo.py line 68 "output = F.softmax(output, dim=1)"  with "output = F.softmax(output)", which solved my issue.

I did not test with v0.3, but switching to it might have been another fix.
